helm/fiftyone-teams-app/README.md

```diff
@@ -37,7 +37,7 @@ Please contact Voxel51 for more information regarding Fiftyone Teams.
   - [From Early Adopter Versions (Versions less than 1.0)](#from-early-adopter-versions-versions-less-than-10)
   - [From Before FiftyOne Teams Version 1.1.0](#from-before-fiftyone-teams-version-110)
   - [From FiftyOne Teams Version 1.1.0 and later](#from-fiftyone-teams-version-110-and-later)
-- [Deploying FiftyOne Teams](#deploying-fiftyone-teams)
+- [Launch FiftyOne Teams](#launch-fiftyone-teams)
```

Comment:
In docker/README.md we use `Deploying FiftyOne Teams`.

Should they be "consistent" and if so, which?

Reply:
I'm good either way - but we should ask the author their intent:
```
❯ git blame -L 394,+1 helm/fiftyone-teams-app/README.md.gotmpl
f06bd484 (Kevin DiMichel 2023-11-10 16:20:45 -0700 394) ## Launch FiftyOne Teams
```
😉

Reply:
Not that guy! : P

I'll change it back by commiting to this PR.

Reply:
Done.